support-frontend/setup.sh

```diff
@@ -148,12 +145,13 @@ main () {
   create_aws_config
   install_node
   install_homebrew
-  install_nginx
-  install_awscli
+  install_brew_deps
+  setup_nginx
   install_jdk
-  install_sbt
-  install_yarn
+  install_yarn_if_linux
   install_js_deps
+
+  EXTRA_STEPS+=("nginx is installed. Ensure you have 'include sites-enabled/*' in your nginx configuration ${NGINX_ROOT}/nginx.conf and add '127.0.0.1 support.thegulocal.com' and '127.0.0.1 support-ui.thegulocal.com' to /etc/hosts")
```

Comment:
this is no longer needed as:
- dev-nginx uses the [`servers` directory](https://github.com/guardian/dev-nginx/blob/master/script/link-config#L25)
- later versions of nginx (certainly any version installed on our new machines) will include the `include servers/*` directive in its default config
- `*.thegulocal.com` exists in DNS to resolve to `127.0.0.1` so we don't need to edits hosts file now